src/bomi/player/mainwindow_m.cpp

```diff
@@ -269,6 +269,19 @@ auto MainWindow::Data::connectMenus() -> void
     connect(play(u"chapter"_q)[u"next"_q], &QAction::triggered, p,
             [seekChapter] () { seekChapter(+1); });
 
+    connect(play[u"state"_q], &QAction::triggered, p, [=] () {
+        QString str;
+        str.append(_MSecToString(e.time()));
+        str.append(u"/"_q);
+        str.append(_MSecToString(e.end()));
+        str.append(u" ("_q);
+        str.append(QString::number(e.time() * 100L / e.end()));
```

Comment:
I think that 1% is enough. I avoided using double because I did not want to deal with rounding errors.